lib/node_modules/@stdlib/utils/homedir/test/test.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var proxyquire = require( 'proxyquire' );
+
+
+// VARIABLES //
+
+var mpath = './../lib/';
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	var homedir;
+	t.ok( true, __filename );
+	homedir = require( mpath );
+	t.strictEqual( typeof homedir, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function aliases `os.homedir` if available', function test( t ) {
+	var homedir;
+	var opts;
+
+	opts = {
+		'os': {
+			'homedir': mock
+		}
+	};
+	homedir = proxyquire( mpath, opts );
+
+	t.strictEqual( homedir, mock, 'equals `os.homedir`' );
+	t.end();
+
+	function mock(){}
+});
+
+tape( 'the function supports older Node versions', function test( t ) {
+	var homedir;
+	var opts;
+	var env;
+
+	opts = {
+		'os': {
+			'homedir': void 0
```

Comment:
Can you confirm that this works as intended? Possible that `proxyquire` ignores `homedir` when set to `undefined`, falling back to the original implementation. If so, can set to `false`.

This should be apparent when generating a coverage report and whether all code branches are covered (notably within the main `index.js` file).